ci/ci/templates/index.html

```diff
@@ -28,7 +28,9 @@ <h2>{{ target.short_str() }}</h2>
             {{ pr.title }}
           </td>
           <td align="left">
+            <a href="{{ pr.last_status }}">
```

Comment:
seems a bit odd that the current build links to the old status?

Reply:
last status is the "last reported". It's the one known as current in the CI.

Reply:
But if it says "Cannot build due to missing image file" but then links to some previously successful build, that seems confusing. I prefer a separate cell that links to the previous build status, if there is any. I also think that link should not appear if `pr.last_status` is `None`.

Reply:
I think it'll be a broken link in that case. I got a bit confused about the meaning of the various statuses -- let's talk tomorrow?